var/spack/repos/builtin/packages/libxi/package.py

```diff
@@ -33,9 +33,11 @@ class Libxi(AutotoolsPackage):
 
     version('1.7.6', 'f3828f9d7893068f6f6f10fe15b31afa')
 
+    depends_on('pkgconfig', type='build')
     depends_on('libx11@1.6:')
     depends_on('libxext@1.0.99.1:')
     depends_on('libxfixes@5:')
+    depends_on('fixesproto@5.0:', type='build')
```

Comment:
add a comment that this is a transient requirement, I guess that should also be taken care of in #4595? @scheibelp ?

Reply:
ACK, will do.